static/app/utils/analytics/workflowAnalyticsEvents.tsx

```diff
@@ -139,7 +138,6 @@ export const workflowEventMap: Record<TeamInsightsEventKey, string | null> = {
   'edit_alert_rule.incompatible_rule': 'Edit Alert Rule: Incompatible Rule',
   'edit_alert_rule.notification_test': 'Edit Alert Rule: Notification Test',
   'issue_alert_rule_details.edit_clicked': 'Issue Alert Rule Details: Edit Clicked',
-  'issue_alert_rule_details.viewed': 'Issue Alert Rule Details: Viewed',
```

Comment:
I noticed that we are still using this in the code, but don't we need this analytics here anymore?

Reply:
the route analytics doesn't use this one i guess

Reply:
can you confirm @scefali ?